apps/evm/src/components/ApproveSteps/ApproveDelegateSteps/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { useTranslation } from 'libs/translations';
+
+import { ApprovalSteps, ApprovalStepsProps } from '..';
+
+export type ApproveDelegateStepsProps = {
+  approveDelegateAction: () => Promise<unknown>;
+  isDelegateApproved: boolean | undefined;
+  isDelegateApprovedLoading: boolean;
+  isApproveDelegateLoading: boolean;
+  hideDelegateApprovalStep?: boolean;
```

Comment:
There are still a few instances here and there in the PR where `delegate` is being used instead of `delegatee`.